atlas/AtlasMl/atlasml/config.py

```
@@ -26,7 +26,7 @@ class Settings(BaseModel):
     @classmethod
     def get_settings(cls):
         """Get the settings from the configuration file."""
-        file_path_env = os.environ.get("APPLICATION_YML_PATH")
+        file_path_env = os.environ.get("APPLICATION_YML_PATH") or str(Path(__file__).parent.parent / "application.yml")
```

Ruff (0.11.9)

Line too long (119 > 88)

(E501)